package.json

```diff
@@ -24,6 +24,9 @@
   },
   "dependencies": {
     "@architect/functions": "^5.3.3",
+    "@aws-sdk/client-cognito-identity-provider": "^3.223.0",
+    "@aws-sdk/client-ses": "^3.259.0",
+    "@aws-sdk/client-sesv2": "^3.223.0",
```

Comment:
These should be dev dependencies, not runtime dependencies, because the packages are preinstalled in the Lambda nodejs18.x runtime. Also, please pin to 3.188.0 because that is the version that is installed in the Lambda runtime.